test/spmd/tensor/parallel/test_tp_examples.py

```diff
@@ -120,6 +121,56 @@ def forward(self, query, key, value):
 
 
 class DistTensorParallelExampleTest(DistTensorTestBase):
+    @with_comms
+    def test_replicate_input_1d(self):
```

Comment:
Nit: you can separate error case and normal cases into two tests. But it's up to you.